tpp-app/src/info/Info.js

```diff
@@ -29,15 +33,42 @@ const LearnMoreCard = () => {
     )
 }
 
-const MenstrualProductCard = ({ onPress, name, image }) =>{
-    return (
+
+
+const MenstrualProductCard = ({name, image, onPress}) =>{
+    return(
         <TouchableOpacity style={styles.productCard} onPress={onPress}>
+        <View>
             <Image style={styles.productIcon} source={image}/>
             <Text style={styles.productText}>{name}</Text>
+        </View>
         </TouchableOpacity>
     )
 }
 
+const getShortenFact = async () => {
+    var factCycle = await GETFactCycle();
+    if (factCycle) {
```

Comment:
i may be tripping but i think what this code does right now is that if fact cycle exists, we enter this if block which then initiates fact cycle again with POSTFactCycle? is this the intended behavior or did you mean to do `if (!factCycle)` annie?

Reply:
I'm tripping, i meant `!factCycle`